lib/models/__tests__/message.test.ts

```diff
@@ -286,3 +292,51 @@ describe("findMessageForRecipient", () => {
     }
   });
 });
+
+describe("attachStoredContent", () => {
+  const aMessageId = "MESSAGE_ID";
+  const aPartitionKey = "PARTITION_KEY";
+  const anAttachmentMeta = { name: "attachmentMeta" };
+  const clientMock = {
```

Comment:
You should instantiate this inside the test as the `upsertAttachment` mock will be not be reset between tests.

As an alternative you can call [resetMock](https://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockreset) on `upsertAttachment` on each test.

Reply:
ok